packages/app/pages/EconomicEvent/EconomicEvent.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+
+import allEconomicEvents from "../../../ui-bindings/EconomicEvent/AllEconomicEvents";
+import economicEventById from "../../../ui-bindings/EconomicEvent/EconomicEventById";
+
+/**
+ * Main component for the page. Contains a search box and a list of
+ * all available Economic Events
+ */
+class EconomicEvent extends React.Component {
+
+  // declare the state and what variables are used
+  state = {
+    eventId: 57
```

Comment:
Why did you choose event 57 here

Reply:
Because it was one that displayed an event with more data attached to it. Should I remove it and set it to null or undefined?
:question:

Reply:
I personally would set it to null.